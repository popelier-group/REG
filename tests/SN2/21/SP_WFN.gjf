# b3lyp/6-311++g(d,p) output=wfn

Single Point Energy

-1 1
 C                 -1.20763500   -0.00006500    0.00017200
 H                 -0.95423800   -0.88952600   -0.55401600
 H                 -0.95416900    0.92460900   -0.49298400
 H                 -0.95431300   -0.03530900    1.04758200
 Br                 1.64339400    0.00001000   -0.00001800
 Cl                -3.21192100    0.00000600   -0.00003500

frame.wfn
