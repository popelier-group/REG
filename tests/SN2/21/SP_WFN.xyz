
6

C       -0.175728    0.384789    1.004272
H       -0.298054   -0.680581    1.116071
H       -0.937610    0.865261    0.411690
H        0.831259    0.701274    0.784510
Br        0.284281   -0.621899   -1.623149
Cl       -0.499474    1.092458    2.851312
