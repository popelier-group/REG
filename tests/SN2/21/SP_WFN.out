 Entering Gaussian System, Link 0=/opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/g16
 Initial command:
 /opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/l1.exe "/scratch/v69787ff/gau_temp_1513317.21/Gau-73128.inp" -scrdir="/scratch/v69787ff/gau_temp_1513317.21/"
 Default is to use a total of   2 processors:
                                2 via shared-memory
                                1 via Linda
 Entering Link 1 = /opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/l1.exe PID=     73129.
  
 Copyright (c) 1988-2019, Gaussian, Inc.  All Rights Reserved.
  
 This is part of the Gaussian(R) 16 program.  It is based on
 the Gaussian(R) 09 system (copyright 2009, Gaussian, Inc.),
 the Gaussian(R) 03 system (copyright 2003, Gaussian, Inc.),
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 16, Revision C.01,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, G. Scalmani, V. Barone, 
 G. A. Petersson, H. Nakatsuji, X. Li, M. Caricato, A. V. Marenich, 
 J. Bloino, B. G. Janesko, R. Gomperts, B. Mennucci, H. P. Hratchian, 
 J. V. Ortiz, A. F. Izmaylov, J. L. Sonnenberg, D. Williams-Young, 
 F. Ding, F. Lipparini, F. Egidi, J. Goings, B. Peng, A. Petrone, 
 T. Henderson, D. Ranasinghe, V. G. Zakrzewski, J. Gao, N. Rega, 
 G. Zheng, W. Liang, M. Hada, M. Ehara, K. Toyota, R. Fukuda, 
 J. Hasegawa, M. Ishida, T. Nakajima, Y. Honda, O. Kitao, H. Nakai, 
 T. Vreven, K. Throssell, J. A. Montgomery, Jr., J. E. Peralta, 
 F. Ogliaro, M. J. Bearpark, J. J. Heyd, E. N. Brothers, K. N. Kudin, 
 V. N. Staroverov, T. A. Keith, R. Kobayashi, J. Normand, 
 K. Raghavachari, A. P. Rendell, J. C. Burant, S. S. Iyengar, 
 J. Tomasi, M. Cossi, J. M. Millam, M. Klene, C. Adamo, R. Cammi, 
 J. W. Ochterski, R. L. Martin, K. Morokuma, O. Farkas, 
 J. B. Foresman, and D. J. Fox, Gaussian, Inc., Wallingford CT, 2019.
 
 ******************************************
 Gaussian 16:  ES64L-G16RevC.01  3-Jul-2019
                11-Nov-2020 
 ******************************************
 --------------------------------
 # b3lyp/6-311++g(d,p) output=wfn
 --------------------------------
 1/38=1,172=1/1;
 2/12=2,17=6,18=5,40=1/2;
 3/5=4,6=6,7=1111,11=2,25=1,30=1,74=-5/1,2,3;
 4//1;
 5/5=2,38=5/2;
 6/7=2,8=2,9=2,10=2,28=1/1;
 99/5=1,6=100,9=1/99;
 -------------------
 Single Point Energy
 -------------------
 Symbolic Z-matrix:
 Charge = -1 Multiplicity = 1
 C                    -1.20764  -0.00007   0.00017 
 H                    -0.95424  -0.88953  -0.55402 
 H                    -0.95417   0.92461  -0.49298 
 H                    -0.95431  -0.03531   1.04758 
 Br                    1.64339   0.00001  -0.00002 
 Cl                   -3.21192   0.00001  -0.00004 
 
                          Input orientation:                          
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0       -1.207635   -0.000065    0.000172
      2          1           0       -0.954238   -0.889526   -0.554016
      3          1           0       -0.954169    0.924609   -0.492984
      4          1           0       -0.954313   -0.035309    1.047582
      5         35           0        1.643394    0.000010   -0.000018
      6         17           0       -3.211921    0.000006   -0.000035
 ---------------------------------------------------------------------
                    Distance matrix (angstroms):
                    1          2          3          4          5
     1  C    0.000000
     2  H    1.078181   0.000000
     3  H    1.078179   1.815161   0.000000
     4  H    1.078185   1.815159   1.815155   0.000000
     5  Br   2.851029   2.801050   2.800934   2.801213   0.000000
     6  Cl   2.004286   2.489035   2.489043   2.489085   4.855315
                    6
     6  Cl   0.000000
 This structure is nearly, but not quite of a higher symmetry.
 Consider Symm=Loose if the higher symmetry is desired.
 Stoichiometry    CH3BrCl(1-)
 Framework group  C1[X(CH3BrCl)]
 Deg. of freedom    12
 Full point group                 C1      NOp   1
 Largest Abelian subgroup         C1      NOp   1
 Largest concise Abelian subgroup C1      NOp   1
                         Standard orientation:                         
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0       -0.175728    0.384789    1.004272
      2          1           0       -0.298054   -0.680581    1.116071
      3          1           0       -0.937610    0.865261    0.411690
      4          1           0        0.831259    0.701274    0.784510
      5         35           0        0.284281   -0.621899   -1.623149
      6         17           0       -0.499474    1.092458    2.851312
 ---------------------------------------------------------------------
 Rotational constants (GHZ):         152.1958929           0.8482273           0.8482273
 Standard basis: 6-311++G(d,p) (5D, 7F)
 There are   126 symmetry adapted cartesian basis functions of A   symmetry.
 There are   121 symmetry adapted basis functions of A   symmetry.
   121 basis functions,   210 primitive gaussians,   126 cartesian basis functions
    31 alpha electrons       31 beta electrons
       nuclear repulsion energy       171.1452452318 Hartrees.
 NAtoms=    6 NActive=    6 NUniq=    6 SFac= 1.00D+00 NAtFMM=   60 NAOKFM=F Big=F
 Integral buffers will be    131072 words long.
 Raffenetti 2 integral format.
 Two-electron integral symmetry is turned on.
 One-electron integrals computed using PRISM.
 NBasis=   121 RedAO= T EigKep=  3.47D-04  NBF=   121
 NBsUse=   121 1.00D-06 EigRej= -1.00D+00 NBFU=   121
 ExpMin= 3.50D-02 ExpMax= 4.40D+05 ExpMxC= 1.51D+04 IAcc=3 IRadAn=         5 AccDes= 0.00D+00
 Harris functional with IExCor=  402 and IRadAn=       5 diagonalized for initial guess.
 HarFok:  IExCor=  402 AccDes= 0.00D+00 IRadAn=         5 IDoV= 1 UseB2=F ITyADJ=14
 ICtDFT=  3500011 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 FoFCou: FMM=F IPFlag=           0 FMFlag=      100000 FMFlg1=           0
         NFxFlg=           0 DoJE=T BraDBF=F KetDBF=T FulRan=T
         wScrn=  0.000000 ICntrl=       500 IOpCl=  0 I1Cent=   200000004 NGrid=           0
         NMat0=    1 NMatS0=      1 NMatT0=    0 NMatD0=    1 NMtDS0=    0 NMtDT0=    0
 Petite list used in FoFCou.
 Keep R1 ints in memory in canonical form, NReq=29147525.
 Requested convergence on RMS density matrix=1.00D-08 within 128 cycles.
 Requested convergence on MAX density matrix=1.00D-06.
 Requested convergence on             energy=1.00D-06.
 No special actions if energy rises.
 SCF Done:  E(RB3LYP) =  -3074.39704367     A.U. after   12 cycles
            NFock= 12  Conv=0.18D-08     -V/T= 2.0014

 **********************************************************************

            Population analysis using the SCF Density.

 **********************************************************************

 Orbital symmetries:
       Occupied  (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A)
       Virtual   (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A)
 The electronic state is 1-A.
 Alpha  occ. eigenvalues -- -482.56749-101.38620 -62.22065 -56.03861 -56.03828
 Alpha  occ. eigenvalues --  -56.03828 -10.08481  -9.29958  -8.42951  -7.06274
 Alpha  occ. eigenvalues --   -7.05523  -7.05523  -6.25319  -6.25193  -6.25193
 Alpha  occ. eigenvalues --   -2.36546  -2.36510  -2.36510  -2.36410  -2.36410
 Alpha  occ. eigenvalues --   -0.67049  -0.54618  -0.49845  -0.30304  -0.30304
 Alpha  occ. eigenvalues --   -0.23699  -0.16071  -0.16071  -0.06492  -0.06403
 Alpha  occ. eigenvalues --   -0.06403
 Alpha virt. eigenvalues --    0.11205   0.11692   0.15150   0.15151   0.17158
 Alpha virt. eigenvalues --    0.17352   0.19435   0.19435   0.20917   0.20917
 Alpha virt. eigenvalues --    0.21356   0.22647   0.22648   0.24804   0.28927
 Alpha virt. eigenvalues --    0.31348   0.31349   0.37232   0.40075   0.54078
 Alpha virt. eigenvalues --    0.54078   0.55808   0.57897   0.59323   0.59323
 Alpha virt. eigenvalues --    0.61872   0.61873   0.68266   0.77062   0.77062
 Alpha virt. eigenvalues --    0.80101   0.82571   0.86779   0.86780   0.87822
 Alpha virt. eigenvalues --    0.87822   0.95709   1.00771   1.00771   1.03418
 Alpha virt. eigenvalues --    1.03419   1.05161   1.15303   1.24926   1.24926
 Alpha virt. eigenvalues --    1.25447   1.55316   1.66242   1.66242   1.82584
 Alpha virt. eigenvalues --    1.88250   1.88250   2.02486   2.19678   2.19679
 Alpha virt. eigenvalues --    2.26531   2.26532   2.32212   2.44859   2.44859
 Alpha virt. eigenvalues --    2.54599   2.63392   2.63591   2.63592   2.71390
 Alpha virt. eigenvalues --    2.71390   3.14082   3.20142   3.86156   3.86157
 Alpha virt. eigenvalues --    4.38525   4.38527   4.39426   4.39427   4.55662
 Alpha virt. eigenvalues --    6.77061   7.79334   7.79344   7.86518   9.83856
 Alpha virt. eigenvalues --   24.04039  25.99344  25.99345  26.43858  48.08154
 Alpha virt. eigenvalues --  215.82246 290.00165 290.00165 290.069951020.84388
          Condensed to atoms (all electrons):
               1          2          3          4          5          6
     1  C    5.489394   0.415536   0.415538   0.415535  -0.046903  -0.147762
     2  H    0.415536   0.516230  -0.037199  -0.037208  -0.009114  -0.038681
     3  H    0.415538  -0.037199   0.516219  -0.037206  -0.009119  -0.038674
     4  H    0.415535  -0.037208  -0.037206   0.516239  -0.009098  -0.038685
     5  Br  -0.046903  -0.009114  -0.009119  -0.009098  35.884202   0.050146
     6  Cl  -0.147762  -0.038681  -0.038674  -0.038685   0.050146  17.383508
 Mulliken charges:
               1
     1  C   -0.541337
     2  H    0.190436
     3  H    0.190443
     4  H    0.190423
     5  Br  -0.860114
     6  Cl  -0.169851
 Sum of Mulliken charges =  -1.00000
 Mulliken charges with hydrogens summed into heavy atoms:
               1
     1  C    0.029964
     5  Br  -0.860114
     6  Cl  -0.169851
 Electronic spatial extent (au):  <R**2>=           1137.2720
 Charge=             -1.0000 electrons
 Dipole moment (field-independent basis, Debye):
    X=             -0.3894    Y=              0.8527    Z=              2.2255  Tot=              2.4148
 Quadrupole moment (field-independent basis, Debye-Ang):
   XX=            -45.1249   YY=            -48.4391   ZZ=            -72.7919
   XY=              1.9156   XZ=              4.9998   YZ=            -10.9360
 Traceless Quadrupole moment (field-independent basis, Debye-Ang):
   XX=             10.3271   YY=              7.0129   ZZ=            -17.3399
   XY=              1.9156   XZ=              4.9998   YZ=            -10.9360
 Octapole moment (field-independent basis, Debye-Ang**2):
  XXX=              5.5171  YYY=            -13.1705  ZZZ=            -56.8393  XYY=              2.0959
  XXY=             -3.1344  XXZ=            -10.5117  XZZ=              6.5252  YZZ=            -14.6206
  YYZ=            -13.3639  XYZ=              2.1724
 Hexadecapole moment (field-independent basis, Debye-Ang**3):
 XXXX=           -109.3317 YYYY=           -242.9289 ZZZZ=          -1321.9497 XXXY=             37.7845
 XXXZ=             99.4321 YYYX=             38.7902 YYYZ=           -222.5333 ZZZX=            119.8098
 ZZZY=           -262.7748 XXYY=            -58.1648 XXZZ=           -221.9379 YYZZ=           -257.4836
 XXYZ=            -72.7877 YYXZ=             35.8927 ZZXY=             21.1789
 N-N= 1.711452452318D+02 E-N=-7.695636395194D+03  KE= 3.070001342532D+03
 Unable to Open any file for archive entry.
 1\1\GINC-NODE333\SP\RB3LYP\6-311++G(d,p)\C1H3Br1Cl1(1-)\ROOT\11-Nov-20
 20\0\\# b3lyp/6-311++g(d,p) output=wfn\\Single Point Energy\\-1,1\C,0,
 -1.207635,-0.000065,0.000172\H,0,-0.954238,-0.889526,-0.554016\H,0,-0.
 954169,0.924609,-0.492984\H,0,-0.954313,-0.035309,1.047582\Br,0,1.6433
 94,0.00001,-0.000018\Cl,0,-3.211921,0.000006,-0.000035\\Version=ES64L-
 G16RevC.01\State=1-A\HF=-3074.3970437\RMSD=1.769e-09\Dipole=-0.950068,
 -0.0000551,0.0001453\Quadrupole=-16.6581663,8.3291021,8.3290642,0.0000
 571,-0.0001982,0.0000068\PG=C01 [X(C1H3Br1Cl1)]\\@
 The archive entry for this job was punched.

 Writing a WFN file to "frame.wfn"


 The chemist is a guest at the physicist's table and
 frequently dines well.
                   -- Richard Bersohn
 Job cpu time:       0 days  0 hours  0 minutes 29.7 seconds.
 Elapsed time:       0 days  0 hours  0 minutes 15.3 seconds.
 File lengths (MBytes):  RWF=     10 Int=      0 D2E=      0 Chk=      2 Scr=      1
 Normal termination of Gaussian 16 at Wed Nov 11 18:51:52 2020.
